### ISSUE TITLE
Util function to manually set a players point prediction

### ISSUE THREAD

**Issue by Tdarnell** [alan-turing-institute/AIrsenal#384]
Similar to the get_player util I would suggest a set_player option, for instances like Sigurdsson where the FPL API hasn't caught up to events and the prediction still see's him as a top points scorer.

**Comment by nbarlowATI** [comment]
Hi @Tdarnell  this is an interesting idea!  There are definitely times when we humans know more about what is likely to happen than we can get from the FPL API! :) 

The other competing philosophy though is to try to minimise the "human in the loop" involvement - there is a case for moving towards having the whole thing run autonomously (including making transfers, and decisions about what chips to play, though we're not yet there with the latter).

For me, the ideal situation would be to combine the best of both worlds - if we could come up with a machine-accessible way of knowing that e.g. a player is very unlikely to play a given match, we could then modify the expected minutes accordingly.   But I think it's not easy! :).    Any thoughts or contributions in this direction would be very welcome!

**Comment by nbarlowATI** [comment]
Yes, this is very true - the bot loves Nat Phillips!   There are a few other players that also get good predictions but that we humans might know better too (I'd be pretty wary of putting Kane in for gw1...)
Perhaps one thing we could do is put in a framework / base class for a "MinutesEstimator" - the default implementation would just do what we do now and look at minutes in previous matches, but people could easily add other derived classes that take e.g. a manual veto-list, or something that scrapes expected line-ups from match previews on the web...?

**Comment by Tdarnell** [comment]
I think a web scraper would be the way to go, but finding the right source
will be the hard part.

**Comment by richardthomasdev** [comment]
While not directly solving the problem, what about an arg that lets the `Teams Selected %` be taken into account?

eg.
`airsenal_make_squad --budget 1000 --min_teams_perc 3`

The above would automatically disqualify both Nat and Siggy from contention, and while not a direct solution might be a cool and less-obtrusive addition it it's own right?

Would love to contribute to this project and take a stab at this, if a direction can be agreed!

**Comment by Tdarnell** [comment]
I do like that approach too, and don't see that as any less AI driven than
web scraping human entered information for predicted lineup's!

**Comment by nbarlowATI** [comment]
@richardthomasdev  that's a very good idea!   Would be great if that could then be applied to the regular week-to-week optimization as well as the initial squad selection, and would be double-great if you could contribute this!! :) 
Feel free to create a new issue to cover that addition (or can just continue here if you prefer), and ask if there's anything we can help with in the implementation!

**Comment by richardthomasdev** [comment]
@nbarlowATI I've opened a separate issue (https://github.com/alan-turing-institute/AIrsenal/issues/400) in case any further discussion directly related to this happens here.
